grails-app/views/occurrence/_map.gsp

```diff
@@ -1145,6 +1207,79 @@
       delete link;
       return false;
     }
+
+    function updateWmsModalContent() {
+        // Get current WMS layer parameters
+        var currentLayer = MAP_VAR.currentLayers[0];
+        if (currentLayer) {
+            var wmsParams = currentLayer.wmsParams;
+            var baseUrl = MAP_VAR.mappingUrl + "/mapping/wms/reflect";
+
+            // Format WMS parameters in tabular format
+            var formattedParams = Object.keys(wmsParams).map(function(key) {
```

Comment:
You can chain map and filter, so something like:

const formattedParams = Object.keys(wmsParams)
    .map(key => (key.toUpperCase() + ':').padEnd(20, ' ') + wmsParams[key])
    .filter(line =>
        !line.startsWith('SIZE:') &&
        !line.startsWith('STYLE:') &&
        !line.startsWith('OUTLINE:') &&
        !line.startsWith('COLOUR:')
    )
    .sort()
    .join('\n');

Reply:
Done